lib/options.nix

```diff
@@ -99,6 +103,33 @@ rec {
     type = lib.types.bool;
   };
 
+  /* Creaties an Option attribute set for an option that specifies the
+     package a module should use.
+
+     The argument default is an attribute set path in pkgs.
+  */
+  mkPackageOption =
+    { name ? last default, pkgs, default ? [ name ], example ? null }:
+    let
+      default' = if !isList default then
+        [ default ]
+      else if head default == "pkgs" then
+        tail default
```

Comment:
this part isn't needed, `pkgs.` can be regarded as implicit.

Reply:
I wanted to make it easier to use so it works if you forget the `pkgs`. I see your point.